cmd/sealer/cmd/cluster/check.go

```diff
@@ -38,7 +38,7 @@ func NewCheckCmd() *cobra.Command {
 		Use:     "check",
 		Short:   "check the state of cluster",
 		Long:    longNewCheckCmdDescription,
-		Example: `sealer check --pre or sealer check --post`,
+		Example: `sealer check --pre \n sealer check --post`,
```

Comment:
I mean change line at "\n", not change or to "\n"

Reply:
sorry, didn't look carefully😢